### ISSUE TITLE
Invalid tag name

### ISSUE THREAD

**Issue by oferze** [wpk-cli/wpk#12]
Did an `npm init` to get an empty project.

Created an empty `index.js` file.

Ran `wpk init` with this output and these answers:

```
warn an error occurred trying to load commands
warn run with --wpk-debug to see the error
wpk init v0.0.3 beta
info webpack v4.28.3 (global)
warn live help disabled (console not tall enough)
warn increase console height to at least 50 rows for live help

Initialising new webpack configuration:
? Config filename: webpack.config.js
? Target execution environment: node
? Context directory path: ./
? Use multiple entries: No
? Entry file: index.js
? Output directory path: dist
? Bundle filename: bundle.js
? Add "~" alias for `context`: No
? Source mapping style: none
? Abort build on errors: never
? Use @babel/polyfill: No
? Add these plugins:
? Add these loaders: Typescript
? Create separate vendor bundle: Yes
? Install external dependencies: Yes
? Save answers for this command as a preset? No
```

Received an error:

```
0 info it worked if it ends with ok
1 verbose cli [ 'C:\\Program Files\\nodejs\\node.exe',
1 verbose cli   'C:\\Users\\ofer\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'install',
1 verbose cli   '--save-dev',
1 verbose cli   '[object',
1 verbose cli   'Object]' ]
2 info using npm@6.4.1
3 info using node@v8.11.3
4 verbose npm-session bd61d0089899de5c
5 silly install loadCurrentTree
6 silly install readLocalPackageData
7 timing stage:rollbackFailedOptional Completed in 1ms
8 timing stage:runTopLevelLifecycles Completed in 32ms
9 verbose stack Error: Invalid tag name "[object": Tags may not have any characters that encodeURIComponent encodes.
9 verbose stack     at invalidTagName (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\node_modules\npm-package-arg\npa.js:87:15)
9 verbose stack     at fromRegistry (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\node_modules\npm-package-arg\npa.js:290:13)
9 verbose stack     at resolve (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\node_modules\npm-package-arg\npa.js:77:12)
9 verbose stack     at npa (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\node_modules\npm-package-arg\npa.js:48:10)
9 verbose stack     at C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\lib\install\deps.js:209:14
9 verbose stack     at C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\node_modules\slide\lib\async-map.js:52:35
9 verbose stack     at Array.forEach (<anonymous>)
9 verbose stack     at C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\node_modules\slide\lib\async-map.js:52:11
9 verbose stack     at Array.forEach (<anonymous>)
9 verbose stack     at asyncMap (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\node_modules\slide\lib\async-map.js:51:8)
9 verbose stack     at exports.getAllMetadata (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\lib\install\deps.js:206:3)
9 verbose stack     at Installer.loadArgMetadata (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\lib\install.js:360:3)
9 verbose stack     at C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\lib\install.js:695:16
9 verbose stack     at BB.join.then (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\lib\install\read-shrinkwrap.js:34:16)
9 verbose stack     at tryCatcher (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\node_modules\bluebird\js\release\util.js:16:23)
9 verbose stack     at Promise._settlePromiseFromHandler (C:\Users\ofer\AppData\Roaming\npm\node_modules\npm\node_modules\bluebird\js\release\promise.js:512:31)
10 verbose cwd c:\Dev\webpack-test
11 verbose Windows_NT 10.0.17134
12 verbose argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\ofer\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save-dev" "[object" "Object]"
13 verbose node v8.11.3
14 verbose npm  v6.4.1
15 error code EINVALIDTAGNAME
16 error Invalid tag name "[object": Tags may not have any characters that encodeURIComponent encodes.
17 verbose exit [ 1, true ]
```

Then, when I ran again with `--wpk-debug`, I got this:

```
err! an error occurred trying to load commands
err! please report at github.com/wpk-cli/wpk/issues
err! error:

  Error: Command failed: npm list -g --depth 0
  npm ERR! peer dep missing: webpack-bundle-analyzer@3.x.x, required by @wpk-cli/command-analyse@0.0.3
  npm ERR! peer dep missing: webpack-dev-server@3.x.x, required by @wpk-cli/command-serve@0.0.3
  npm ERR! peer dep missing: webpack-bundle-analyzer@3.x.x, required by @wpk-cli/command-analyse@0.0.3
  npm ERR! missing: npmlog@, required by npm-registry-client@7.0.8

```
Then a list of packages and then:

```
      at makeError (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\execa\index.js:172:9)
      at Function.module.exports.sync (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\execa\index.js:341:15)
      at C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\dist\command-find.js:57:36
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
      at _next (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\@babel\runtime\helpers\asyncToGenerator.js:25:9)
      at C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\@babel\runtime\helpers\asyncToGenerator.js:32:7
      at Promise._execute (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\bluebird\js\release\debuggability.js:313:9)
      at Promise._resolveFromExecutor (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\bluebird\js\release\promise.js:483:18)
      at new Promise (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\bluebird\js\release\promise.js:79:10)
      at C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\@babel\runtime\helpers\asyncToGenerator.js:21:12
      at _findCommands (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\dist\command-find.js:123:24)
      at findCommands (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\dist\command-find.js:41:24)
      at C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\dist\cli.js:163:91
      at Generator.next (<anonymous>)
      at asyncGeneratorStep (C:\Users\ofer\AppData\Roaming\npm\node_modules\@wpk-cli\cli\node_modules\@babel\runtime\helpers\asyncToGenerator.js:3:24)
```

My env:

Node - v8.11.3
npm - 6.4.1
webpack -4.28.3
Windows 10

**Comment by sdgluck** [comment]
Looks like this is related to #11. Will look at putting in a fix asap. Thanks for the issue!

**Comment by sdgluck** [comment]
@oferze Fix is in for this on latest (0.0.4). Thanks!

**Comment by oferze** [comment]
Yep, can confirm it's working now, thanks!
